test/suites/zombie_tanssi_relay/test_zombie_tanssi_relay.ts

```diff
@@ -13,17 +10,24 @@ import {
     signAndSendAndInclude,
     waitSessions,
 } from "utils";
+import { addApiMonitoringToContext } from "../../utils/block-production-monitor.ts";
+import type { AuthorityAssignmentCollatorContainerChain } from "../../types/authority-assignment-collator-container-chain.ts";
+import type { AuthorNotingAuthor } from "../../types/author-noting-author.ts";
+import type { GenesisData } from "../../types/genesis-data.ts";
+import type { Wallet } from "ethers";
 
 describeSuite({
     id: "ZOMBIETANSS01",
     title: "Zombie Tanssi Relay Test",
     foundationMethods: "zombie",
     testCases: ({ it, context }) => {
+        addApiMonitoringToContext(context);
+
         let relayApi: ApiPromise;
         let container2000Api: ApiPromise;
         let container2001Api: ApiPromise;
         let container2002Api: ApiPromise;
-        let ethersSigner: Signer;
+        let ethersSigner: Wallet;
```

Comment:
All changes below in this file are related to the fixing types, after `@ts-nocheck` is removed.